app/src/main/java/com/example/bigowlapp/HomeActivity.java

```diff
@@ -61,7 +62,7 @@ public void onClick(View v) {
             btnMonitoringGroup.setOnClickListener(new View.OnClickListener() {
                 @Override
                 public void onClick(View v) {
-                    Intent i = new Intent(HomeActivity.this, MonitoringGroupPageActivity.class);
+                    Intent i = new Intent(HomeActivity.this, MonitoringGroupListActivity.class);
```

Comment:
Change the intent back to MonitoringGroupPageActivity.class as we only allowed 1 monitoring group per user for now. Keep the MonitoringGroupListActivity file for future possible feature.